### ISSUE TITLE
Parsing error: parser.parse is not a function

### ISSUE THREAD

**Issue by silverbucket** [codeclimate/codeclimate-eslint#512]
# Bug Template

## Actual Behavior

On codeclimate.com the PR is failing but the only error message shown on the site is `Parsing error: parser.parse is not a function`

## Expected Behavior

If there is an actual issue, it should be explained in an understandable way. However using the codeclimate CLI no errors are reported.

## Steps to reproduce

> 1. _Include configurations used in your: <br>
> (1) .codeclimate.yml <br>
> (2) config files for the oss analysis tool._<br>

> 2. _If possible, use our CLI with [the debug option](https://github.com/codeclimate/codeclimate#environment-variables) to provide additional info, including any stderr produced by the engine._ `CODECLIMATE_DEBUG=1 codeclimate analyze -e eslint`

See the PR here: https://github.com/sockethub/sockethub/pull/325

Using the CLI, no errors shown. 

## Relevant links

----
- [ ] 4-6 word issue title 

https://github.com/sockethub/sockethub/pull/325

**Comment by efueger** [comment]
Hi @silverbucket - thanks for the issue and mention [here](https://github.com/gilbarbara/codeclimate-stylelint/issues/33#issuecomment-778470025) 👍🏼 

You're likely seeing that parsing error because of the default version of ESLint (3.19.0) used in our in-app plugin settings.

To [use a different eslint version](https://docs.codeclimate.com/docs/eslint#eslint-versions), you'll need to add a .codeclimate.yml with something like this:

- https://github.com/efueger/sockethub/blob/eslint-consolidate/.codeclimate.yml

Let me know if you bump into anything else! Here to help.

**Comment by silverbucket** [comment]
Thank you for the info @efueger, this solved the problem.

**Comment by zavoloklom** [comment]
Maybe it would be helpful.

I got error:
```
D, [2021-03-03T22:16:51.949006 #1] DEBUG -- : engine stderr: •• Timing: .engineConfig: 2.192s
D, [2021-03-03T22:16:54.717900 #1] DEBUG -- : engine stderr: Error: parser.parse is not a function
D, [2021-03-03T22:16:54.718042 #1] DEBUG -- : engine stderr: See our documentation at https://docs.codeclimate.com/docs/stylelint for more information.
I, [2021-03-03T22:16:55.157093 #1]  INFO -- : finished engine stylelint
error: (CC::CLI::Analyze::EngineFailure) engine stylelint failed with status 1 and stderr 
•• Timing: .engineConfig: 2.192s
Error: parser.parse is not a function
See our documentation at https://docs.codeclimate.com/docs/stylelint for more information.
```

I don't why it helps but in my case it was missing `.babelrc` file. When I added it - error has gone.
